roles/restic/templates/restic.service.j2

```diff
@@ -17,7 +17,12 @@ ExecStart=/bin/sh -c '{{ commands.command }} | {{ restic_binary }} backup --verb
 {% endfor %}
 
 {% if restic_backup_paths | length > 0 %}
-ExecStart={{ restic_binary }} --verbose backup {{ restic_backup_paths | join(' ') }}
+ExecStart={{ restic_binary }} backup {{ restic_backup_paths | join(' ') }}{% if restic_backup_parameters | length > 0 %} \
+    {% for param in restic_backup_parameters | dict2items %}
+    --{{ param.key }}{% if param.value %}={{ param.value }}{% endif %}{% if not loop.last %} \{% endif %}
```

Comment:
This should be extended to loop over lists if there's a list of values, so that you can exclude multiple paths for example.

Reply:
Done. I do not wish to speak about jinja2's whitespace control.